src/flow/darcy_flow_mh.cc

```diff
@@ -600,10 +600,8 @@ bool DarcyMH::zero_time_term(bool time_global) {
     }
 }
 
-Vec DarcyMH::compute_full_residual_vec()
+Vec DarcyMH::compute_full_residual_vec(Vec residual_)
```

Comment:
Now there is no need for return value of this method. It can be declared as void and the return statement omitted.